Demo/Demo/Info.plist

```diff
@@ -2,6 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIAppFonts</key>
+	<array>
+		<string>PayPalOpen-Regular.otf</string>
+		<string>PayPalOpen-Medium.otf</string>
+		<string>PayPalOpen-Bold.otf</string>
```

Comment:
qq: Do we need both Medium and Bold fonts? Checking to see if we could save merchants a few bytes.

Reply:
They aren't needed for the redesign, so I'll remove